### PR TITLE
Improve profile page actions layout

### PR DESCRIPTION
**Pull request by ryangjchandler**
- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

Changes the layout of the default "Profile" page. Original design had an action link which looked out of place.

New design in screenshot below.

CleanShot 2023-08-15 at 11 43 22@2x

Also introduced a new `getBackFormAction()` method for consistency with the `getSaveFormAction()` method.
Marked `backAction()` method as deprecated, pointing to new method as replacement.

Added `Alignment::Between` case and support for that inside of the actions component.

### CODE REVIEW

**Comment by zepfietje** [comment]
Personally, I like right-aligned actions, but since we're left-aligning them in our design system, I think it's best if we do so for these pages too.

Makes them match modals and looks ok if you change the "Back" label to "Cancel" (just like we have for modals and resource edit pages).

**Comment by danharrin** [comment]
@zepfietje what needs to be done here to get it ready for merge?

**Comment by zepfietje** [comment]
> @zepfietje what needs to be done here to get it ready for merge?

1. Change `Back` to `Cancel`.
2. Align buttons to start, so save button comes before cancel.
3. Make sure buttons are **not** full-width.

Request my review when those changes have been implemented (by whoever is working on it now 😅) and I'll merge it. 😄

**Comment by ryangjchandler** [comment]
@zepfietje 
CleanShot 2023-08-17 at 10 08 28@2x

File: packages/panels/src/Pages/Auth/EditProfile.php [hunk 1]
```diff
@@ -269,7 +275,12 @@ protected function getSaveFormAction(): Action
 
     protected function hasFullWidthFormActions(): bool
     {
-        return true;
+        return false;
+    }
+
+    public function getFormActionsAlignment(): string
```
Comment (danharrin) [inline packages/panels/src/Pages/Auth/EditProfile.php]:
```suggestion
    public function getFormActionsAlignment(): string | Alignment
```

Comment (danharrin) [reply packages/panels/src/Pages/Auth/EditProfile.php]: Might need to update a few other places

File: packages/actions/resources/views/components/actions.blade.php [hunk 2]
```diff
@@ -27,6 +27,7 @@
                     match ($alignment) {
                         Alignment::Center, 'center' => 'justify-center',
                         Alignment::End, Alignment::Right, 'end', 'right' => 'flex-row-reverse',
+                        Alignment::Between, 'between' => 'justify-between',
```
Comment (zepfietje) [inline packages/actions/resources/views/components/actions.blade.php:30]: We no longer need this, right?

File: packages/panels/resources/lang/en/pages/auth/edit-profile.php [hunk 3]
```diff
@@ -43,7 +43,7 @@
     'actions' => [
 
         'back' => [
```
Comment (zepfietje) [inline packages/panels/resources/lang/en/pages/auth/edit-profile.php]: Not sure if we can somehow change the `back` key to `cancel` in a non-breaking way?

Comment (danharrin) [reply packages/panels/resources/lang/en/pages/auth/edit-profile.php]: We can just find and replace

Comment (zepfietje) [reply packages/panels/resources/lang/en/pages/auth/edit-profile.php]: Wouldn't that break applications where people override this translation string?

Comment (danharrin) [reply packages/panels/resources/lang/en/pages/auth/edit-profile.php]: We haven't been considering that as a breaking change up until this point

Comment (zepfietje) [reply packages/panels/resources/lang/en/pages/auth/edit-profile.php]: Fine.

**Pull request merged**